cran_gen/volume.h

```diff
@@ -49,6 +49,7 @@ NT volume(Polytope &P,
     typedef Ball<Point> Ball;
     typedef BallIntersectPolytope<Polytope,Ball> BallPoly;
     typedef typename Parameters::RNGType RNGType;
+    typedef typename Polytope::VT VT;
```

Comment:
the whole file should not be here, right ?

Reply:
it is the cran version file because it uses RcppEigen and BH packages.